classes/oembed/class-oembed.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Automattic\Gravatar\GravatarEnhanced\OEmbed;
+
+class OEmbed {
+	private const API_ENDPOINT = 'https://api.gravatar.com/v3/oembed';
+
+	/**
+	 * @return void
+	 */
+	public function init() {
+		add_action( 'init', [ $this, 'register_oembed_provider' ] );
```

Comment:
I'm not sure we need to register it in `init` - I think it can just be registered. Makes no difference either way though!

Reply:
✅ You are right! Have updated the code. Thanks!